api/internal/localizer/util.go

```diff
@@ -99,3 +101,7 @@ func hasRef(repoURL string) bool {
 	}
 	return repoSpec.Ref != ""
 }
+
+func isBuiltinPlugin(res *resource.Resource) bool {
+	return res.GetGvk().Group == "" && res.GetGvk().Version == konfig.BuiltinPluginApiVersion
```

Comment:
`res.GetGvk` is a nontrivial operation, we should put it in a variable rather than calling it twice.

Reply:
I changed this in the existing implementation.